src/main/java/seedu/address/commons/util/JsonUtil.java

```diff
@@ -37,7 +37,8 @@
             .setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY)
             .registerModule(new SimpleModule("SimpleModule")
                     .addSerializer(Level.class, new ToStringSerializer())
-                    .addDeserializer(Level.class, new LevelDeserializer(Level.class)));
+                    .addDeserializer(Level.class, new LevelDeserializer(Level.class))
+                    .addSerializer(Path.class, new ToStringSerializer()));
```

Comment:
what is this change for?

Reply:
the new jackson module i used serialises paths as absolute paths rather than relative paths, causing it to fail tests. this reverts it back to relative paths